qemu/tests/migration_multi_host.py

```diff
@@ -48,6 +52,7 @@ def __init__(self, test, params, env):
             self.mig_bg_kill_command = params.get("migration_bg_kill_command",
                                                   "pkill -9 ping")
             self.need_to_login = params.get("need_to_login", "no")
+            self.mig_protocol = mig_protocol
```

Comment:
As we implement the class for MultihostMigrationRdma we don't need this any more